dist/dialog/ds6/dialog.css

```diff
@@ -0,0 +1,168 @@
+}
+.dialog__window--fill {
+  min-height: 70%;
+}
+.dialog__window--full {
+  max-width: 100%;
+  width: 100%;
+  bottom: 0;
+}
+.dialog__window--left,
+.dialog__window--right {
+  width: calc(88% - 7px);
+  min-width: calc(88% - 7px);
+  max-width: calc(100% - 7px);
+}
+.dialog__window--left {
+  right: auto;
+  border-right: 1px solid rgba(153, 153, 153, 0.18);
+}
+.dialog__window--right {
+  left: auto;
+  border-left: 1px solid rgba(153, 153, 153, 0.18);
+}
+.dialog__mask {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  opacity: 0.7;
+  background-color: #111820;
+  will-change: opacity;
+}
+.dialog__body {
+  padding: 16px;
+  width: 100%;
+  min-height: 100%;
+  box-sizing: border-box;
+}
+.dialog__body :first-child {
+  margin-top: 0;
+  padding-top: 0;
+}
+.dialog__body :last-child {
+  margin-bottom: 0;
+  padding-bottom: 0;
+}
+.dialog__close {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 16px;
+  border: 0;
+  cursor: pointer;
+  color: #111820;
+  background: #ffffff;
+}
+.dialog__close:hover,
+.dialog__close:focus {
+  color: #006efc;
+}
+.dialog--show .dialog__window--fade,
+.dialog--hide .dialog__window--fade,
+.dialog--show .dialog__mask--fade,
+.dialog--hide .dialog__mask--fade {
+  transition: opacity 0.16s ease-out;
+}
+.dialog--show .dialog__window--slide,
+.dialog--hide .dialog__window--slide {
+  transition: -webkit-transform 0.32s ease-out;
+  transition: transform 0.32s ease-out;
+  transition: transform 0.32s ease-out, -webkit-transform 0.32s ease-out;
+}
+.dialog--show .dialog__mask--fade-slow,
+.dialog--hide .dialog__mask--fade-slow {
+  transition: opacity 0.32s ease-out;
+}
+.dialog--hide .dialog,
+.dialog--show--from .dialog {
+  display: block;
+}
+.dialog--hide .dialog__window--fade,
+.dialog--show--from .dialog__window--fade,
+.dialog--hide .dialog__mask--fade,
+.dialog--show--from .dialog__mask--fade,
+.dialog--hide .dialog__mask--fade-slow,
+.dialog--show--from .dialog__mask--fade-slow {
+  opacity: 0;
+}
+.dialog--hide .dialog__window--slide.dialog__window--left,
+.dialog--show--from .dialog__window--slide.dialog__window--left {
+  -webkit-transform: translateX(-100%);
+          transform: translateX(-100%);
+}
+.dialog--hide .dialog__window--slide.dialog__window--right,
+.dialog--show--from .dialog__window--slide.dialog__window--right {
+  -webkit-transform: translateX(100%);
+          transform: translateX(100%);
+}
+.dialog--show .dialog,
+.dialog--hide--from .dialog {
+  display: block;
+}
+.dialog--show .dialog__window--fade,
+.dialog--hide--from .dialog__window--fade {
+  opacity: 1;
+}
+.dialog--show .dialog__window--slide,
+.dialog--hide--from .dialog__window--slide {
+  -webkit-transform: translateX(0);
+          transform: translateX(0);
+}
+.dialog--show .dialog__mask--fade,
+.dialog--hide--from .dialog__mask--fade,
+.dialog--show .dialog__mask--fade-slow,
+.dialog--hide--from .dialog__mask--fade-slow {
+  opacity: 0.7;
+}
+@media (min-width: 601px) {
```

Comment:
After talking with @senthilp we came to the conclusion that we would only create separate flagged bundles when the css is somewhat substantially different. In this case since it's just a few lines of CSS it seemed to make sense to keep it in one bundle.

Reply:
Ok, that's fine.